lib/internal/Magento/Framework/DB/Logger/LoggerProxy.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\DB\Logger;
+
+class LoggerProxy extends LoggerAbstract
+{
+    /**
+     * Logger alias param name
+     */
+    const PARAM_ALIAS = 'db_logger_mode';
```

Comment:
please group Deployment configuration options related to logging together and rename them:

'db_logger' => [
     'output' => 'file',
     'log_everything' => 1,
     'query_time_threshold' => '0.001',
     'include_stacktrace' => 1
]